frame/support/src/weights.rs

```diff
@@ -709,6 +712,26 @@ where
 	}
 }
 
+/// Implementor of [`WeightToFeePolynomial`] that uses a constant modifier.
+pub struct ConstantModifierFee<T, M>(sp_std::marker::PhantomData<(T, M)>);
```

Comment:
This basically multiplies the length by `M`, how is the name conveying this? I don't quite get it. Perhaps add a single line rustdoc here with one example to help confused readers :)) 

I would've called it `ConstantMultiplier<T, M>`.

Reply:
I struggled with the name as well, but moreso because `Constant` isn't strictly correct :)

I renamed it based on your suggestion.